.fixtures.yml

```diff
@@ -27,7 +27,7 @@ fixtures:
     ruby_task_helper: 'https://git@github.com/puppetlabs/puppetlabs-ruby_task_helper'
     cron_core: 'https://github.com/puppetlabs/puppetlabs-cron_core'
     bootstrap: 'https://github.com/puppetlabs/puppetlabs-bootstrap'
-    node_manager: 'https://github.com/WhatsARanjit/puppet-node_manager'
+    node_manager: 'https://github.com/MartyEwings/puppet-node_manager.git'
```

Comment:
I don't think that's a good idea. As far as I know the fork from @MartyEwings isn't on the forge and people will probably not switch to it. Instead we should wait until https://github.com/WhatsARanjit/puppet-node_manager/pull/71 is merged.

Reply:
@bastelfreak this isnt a dependency of the module, its a dependency of our CI for nightly builds of PE.

This is an operational decision to make our nightly workflows work, which are a priority for PE Puppet 8 development

Reply:
Iv highlighted that the PEADM team (the module we use for PE deployment in CI) need to either remove this dependency or work with Ranjit to get https://github.com/WhatsARanjit/puppet-node_manager/pull/71 merged ASAP